seminar_registration_system/member_info_capture/urls.py

```diff
@@ -1,8 +1,21 @@
-from django.urls import path
-from . import views
+from django.urls import path, include
+from member_info_capture import views
+from django.views.generic import TemplateView
+from rest_framework import routers
+from .views import MemberViewSet, MemberInfoCaptureViewSet
 
+
+router = routers.DefaultRouter()
+router.register('member', MemberViewSet)
+router.register('member_info_capture', MemberInfoCaptureViewSet)
 urlpatterns = [
+    path('api/', include(router.urls)),
```

Comment:
file `urls.py` import the `MemberInfoCaptureViewSet` and create a router to register the viewset